@CodeAbyss Thanks for the PR. There is failure in CI test case

```
03:14 The full traceback is:
03:14 Traceback (most recent call last):
03:14   File "/root/.ansible/tmp/ansible-tmp-1550575112.41-156319241715644/AnsiballZ_pids.py", line 128, in <module>
03:14     _ansiballz_main()
03:14   File "/root/.ansible/tmp/ansible-tmp-1550575112.41-156319241715644/AnsiballZ_pids.py", line 120, in _ansiballz_main
03:14     invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)
03:14   File "/root/.ansible/tmp/ansible-tmp-1550575112.41-156319241715644/AnsiballZ_pids.py", line 63, in invoke_module
03:14     imp.load_module('__main__', mod, module, MOD_DESC)
03:14   File "/tmp/ansible_pids_payload_kgIbrv/__main__.py", line 74, in <module>
03:14   File "/tmp/ansible_pids_payload_kgIbrv/__main__.py", line 69, in main
03:14   File "/tmp/ansible_pids_payload_kgIbrv/__main__.py", line 56, in get_pid
03:14 AttributeError: 'NoneType' object has no attribute 'lower'
03:14 
03:14 fatal: [testhost]: FAILED! => {
03:14     "changed": false, 
03:14     "module_stderr": "Traceback (most recent call last):\n  File \"/root/.ansible/tmp/ansible-tmp-1550575112.41-156319241715644/AnsiballZ_pids.py\", line 128, in <module>\n    _ansiballz_main()\n  File \"/root/.ansible/tmp/ansible-tmp-1550575112.41-156319241715644/AnsiballZ_pids.py\", line 120, in _ansiballz_main\n    invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)\n  File \"/root/.ansible/tmp/ansible-tmp-1550575112.41-156319241715644/AnsiballZ_pids.py\", line 63, in invoke_module\n    imp.load_module('__main__', mod, module, MOD_DESC)\n  File \"/tmp/ansible_pids_payload_kgIbrv/__main__.py\", line 74, in <module>\n  File \"/tmp/ansible_pids_payload_kgIbrv/__main__.py\", line 69, in main\n  File \"/tmp/ansible_pids_payload_kgIbrv/__main__.py\", line 56, in get_pid\nAttributeError: 'NoneType' object has no attribute 'lower'\n", 
03:14     "module_stdout": "", 
03:14     "msg": "MODULE FAILURE\nSee stdout/stderr for the exact error", 
03:14     "rc": 1
03:14 }

```

--- a/lib/ansible/modules/system/pids.py
+++ b/lib/ansible/modules/system/pids.py
@@ -53,7 +53,7 @@ except ImportError:
 
 
 def get_pid(name):
-    return [p.info['pid'] for p in psutil.process_iter(attrs=['pid', 'name']) if bool(name) and name.lower() == p.info['name'].lower()]
+    return [p.info['pid'] for p in psutil.process_iter(attrs=['pid', 'name']) if p.info and p.info.get('name', None) and p.info['name'].lower() == name.lower()]
 
 
 def main():

--- a/lib/ansible/modules/system/pids.py
+++ b/lib/ansible/modules/system/pids.py
@@ -53,7 +53,7 @@ except ImportError:
 
 
 def get_pid(name):
-    return [p.info['pid'] for p in psutil.process_iter(attrs=['pid', 'name']) if name.lower() == p.info['name'].lower()]
+    return [p.info['pid'] for p in psutil.process_iter(attrs=['pid', 'name']) if bool(name) and name.lower() == p.info['name'].lower()]
 
 
 def main():

--- a/lib/ansible/modules/system/pids.py
+++ b/lib/ansible/modules/system/pids.py
@@ -53,7 +53,7 @@ except ImportError:
 
 
 def get_pid(name):
-    return [p.info['pid'] for p in psutil.process_iter(attrs=['pid', 'name']) if p.info and p.info.get('name', None) and p.info['name'].lower() == name.lower()]
+    return [ p.info['pid'] for p in psutil.process_iter(attrs=['pid', 'name']) if p.info and p.info.get('name', None) and p.info['name'].lower() == name.lower() ]
 
 
 def main():

